lib/node_modules/@stdlib/stats/base/dists/signrank/cdf/benchmark/benchmark.js

```diff
@@ -31,16 +31,23 @@ var cdf = require( './../lib' );
 // MAIN //
 
 bench( pkg, function benchmark( b ) {
-	var n;
+	var nValues;
+	var values;
```

Comment:
```suggestion
	var len;
	var n;
```

Please add only `len` and keep the original variable names unchanged (for consistency).